### ISSUE TITLE
Not compatible with 1.19.3

### ISSUE THREAD

**Issue by MeeniMc** [CraftyCorvid/ImprovedSigns#28]
Here is the error

```
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'improved-signs'! 
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
 	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) 
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
 	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) 
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52) 
	at net.minecraft.class_310.<init>(class_310.java:465)
 	at net.minecraft.client.main.Main.method_44604(Main.java:205)
 	at net.minecraft.client.main.Main.main(Main.java:51)
 	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
 	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
 	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
 	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
 	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) 
Caused by: java.lang.NoSuchFieldError: field_25299
 	at com.ivanff.improvedSigns.loot.condition.LootConditionTypes.register(LootConditionTypes.java:13)
 	at com.ivanff.improvedSigns.ImprovedSignsMod.onInitialize(ImprovedSignsMod.java:26)
 	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
 	... 11 more
```

**Comment by EGOIST1372** [comment]
I love this mod, I hope we get an update for 1.19.,3
thanks in advance <3

**Comment by sternschnaube** [comment]
I would love to see it available for 1.19.3 👍🏿

**Comment by MeeniMc** [comment]
Thanks for updating ❤️
